### ISSUE TITLE
text classification error

### ISSUE THREAD

**Issue by ruin2** [kpe/bert-for-tf2#28]
![image](https://user-images.githubusercontent.com/45546580/70962459-47490980-20c0-11ea-8b98-4118e6ab7793.png)

i followed your codes to write a ten class textclassification and failed (output predictions are the same) , logits seems to be equal correspondingly. i dont konw why ,and ask for helps, thank you

my model is just that simple:
lbert(tokens)
dense(10)

but after i forze bert layers , and add some additional layers like lstm , it worked,well , accuracy is still under my expectation..

SORRY!
its my mistacks!
i wrote a wrong sequnces padder and a wrong tokenizer....
emmmmmmmmmmmmm

**Comment by kpe** [comment]
sequence classification with BERT should work well enough without an LSTM. As an example check https://github.com/kpe/bert-for-tf2/blob/master/examples/tpu_movie_reviews.ipynb
